### DISCUSSION TITLE
AR unsupported

### DISCUSSION THREAD

**Discussion by Asajco** [pmndrs/xr#310] [General] [answered]
Hello there, i just copied one of examples how to use AR, but on my localhost i'm getting AR unsupported even i enabled AR in Chrome, its same in Safari. Any help how to resolve this? Thanks a lot :)

**Comment by bbohlender** [comment, marked as answer]
What device are you using to open the example? A smartphone, VR glasses, or something else?
Check out the official WebXR examples and see if they work https://immersive-web.github.io/webxr-samples/immersive-vr-session.html
